.travis.yml

```diff
@@ -26,8 +26,9 @@ install:
 - npm install
 before_script:
 # Install Firefox Nightly to prevent signing errors.
+- npm i -g get-firefox
```

Comment:
No need to adjust the path, I'd just have to call it from `node_modules/.bin/get-firefox`, however it is weird to have it as dep then.